types/protractor-beautiful-reporter/index.d.ts

```diff
@@ -0,0 +1,65 @@
+// Type definitions for Angularized HTML Reporter with Screenshots for Protractor 1.3
+// Project: https://github.com/Evilweed/protractor-beautiful-reporter
+// Definitions by: Adam Kwiatek <https://github.com/akwiatek>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import jasmine = require('jasmine');
+
+export = HtmlReporter;
+
+declare class HtmlReporter {
+    constructor(options: HtmlReporter.HtmlReporterConstructorOptions);
+    getJasmine2Reporter(): jasmine.CustomReporter;
+}
+
+declare namespace HtmlReporter {
+    interface HtmlReporterConstructorOptions {
```

Comment:
should it accept `jasmine2MetaDataBuilder` option?:
https://github.com/Evilweed/protractor-beautiful-reporter#jasmine2-meta-data-builder-optional

Reply:
Done